app/src/main/res/values/strings-untranslated.xml

```diff
@@ -6,5 +6,5 @@
     <!-- A collection of strings that do not need to be translated (e.g. brand names, features that are en-US only, etc.) -->
 
     <!-- Title text shown above the Pocket channel -->
-    <string name="pocket_channel_title" translatable="false">Pocket Editor\'s Choice</string>
+    <string name="pocket_channel_title" translatable="false">Recommended by Pocket</string>
```

Comment:
nit: I wonder if we should substitute the brand name into this String. Since it's untranslated, it's probably not important though if you wanted to move on to different things.